label_sleuth/config.json

```diff
@@ -1,11 +1,11 @@
 {
-    "first_model_positive_threshold": 20,
```

Comment:
This was probably committed by mistake. We don't want to change these settings

Reply:
Indeed, I will remove that change from the commit. Good catch